validity/src/proposer.rs

```diff
@@ -106,6 +105,17 @@ where
         // Initialize fetcher
         let rollup_config_hash = hash_rollup_config(fetcher.rollup_config.as_ref().unwrap());
 
+        let gas_threshold = env::var("GAS_THRESHOLD")
```

Comment:
Don't pick the env like this here, take a look to the `env.rs` add it there and set a default value.

Reply:
We don't want a default value, as not setting this will make it default to the normal behavior.